policy-controller/k8s/index/src/outbound/index.rs

```diff
@@ -1336,7 +1440,37 @@ pub fn is_parent_service_or_egress_network(parent: &ParentReference) -> bool {
 }
 
 #[inline]
-fn route_accepted_by_parent(
+fn route_accepted_by_resource_port(
+    route_status: Option<&k8s_gateway_api::RouteStatus>,
+    resource_port: &ResourcePort,
+) -> bool {
+    let (kind, group) = match resource_port.kind {
+        ResourceKind::Service => ("Service", "core"),
+        ResourceKind::EgressNetwork => ("EgressNetwork", "policy.linkerd.io"),
+    };
```

Comment:
It would be ideal to use the type-oriented helpers
```suggestion
    let (kind, group) = match resource_port.kind {
        ResourceKind::Service => (Service::kind(&()), Service::group(&())),
        ResourceKind::EgressNetwork => (linkerd_k8s_api::EgressNetwork::kind(&()), linkerd_k8s_api::EgressNetwork::group(&())),
    };
```

So that we have more typo-proof static checking.